CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to the CLAW Compiler project are documented in this file.
 
+## [2.0.2] - 2020-07-27
+* OMNI Compiler submodule now pointing to
+  omni-compiler/xcodeml-tools@3bbc8ec4caa31113a7f67ae7371e678c28d613fc
+* Added support for NAG compiler
+* Various bug fixes
```

Comment:
Maybe you could make clear that these are changes coming from the updated omni, not CLAW itself

Reply:
As far as I understand, they are actually coming from CLAW. There were various commits since 2.0.1 release.

Reply:
You're totally right, should have checked the history.